docs/tutorials/geomesa-examples-gdelt.rst

```diff
@@ -12,7 +12,7 @@ Prerequisites
 
 .. warning::
 
-    You will need access to a Hadoop 2.2.2 or better
+    You will need access to a Hadoop |hadoop_version| or better
```

Comment:
Would it be better to change `hadoop_version` to be `minimum_hadoop_version` or something similar and only have it be a version number?  With that, we can write sentences like "GeoMesa requires at least Hadoop version `mhv`" and "`mhv` or later"...